sources/packages/backend/libs/services/src/queue/queue.module.ts

```diff
@@ -67,9 +68,15 @@ async function getConnectionFactory(
  * Builds the bull module options to register queues in a dynamic way.
  * @returns bull module options with all existing queues.
  */
-function getQueueModules(): BullModuleOptions[] {
-  return Queues.map<BullModuleOptions>((queue) => ({
+function getQueueModules(): BullModuleAsyncOptions[] {
```

Comment:
What was the motivation to change it to async? Is it in preparation for a possible future change?

Reply:
We need async to include the config service from where the queue prefix come from.